<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebateWindow</class>
 <widget class="QWidget" name="DebateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">     background-color:white;
</string>
  </property>
  <widget class="QPushButton" name="playerButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>112</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="opponentButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>112</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>419</y>
     <width>1000</width>
     <height>275</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="answerCButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
     background-color:rgba(0, 145, 255, 128);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerDButton">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>180</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
     background-color:rgba(0, 145, 255, 128);
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerBButton">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>90</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
     background-color:rgba(0, 145, 255, 128);
}</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerAButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
     background-color:rgba(0, 145, 255, 128);
}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>700</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">     background-color:rgba(242, 242, 242, 0);
</string>
    </property>
    <property name="text">
     <string>Question: Enter Question Here</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerALabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>350</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerCLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>350</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerBLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>350</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerDLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>350</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextQuestionButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
     background-color:rgba(0, 145, 255, 128);
}</string>
    </property>
    <property name="text">
     <string>Next Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnToMenuButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>125</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
    </property>
    <property name="text">
     <string>Game Menu</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="myVotesBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>30</width>
      <height>160</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="oppVotesBar">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>30</width>
      <height>160</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="myVotesScore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 0)</string>
    </property>
    <property name="text">
     <string>0000000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>255</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 0)</string>
    </property>
    <property name="text">
     <string>My Votes</string>
    </property>
   </widget>
   <widget class="QLabel" name="oppVotesScore">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 0)</string>
    </property>
    <property name="text">
     <string>0000000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>255</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 0)</string>
    </property>
    <property name="text">
     <string>Opponent Votes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
